.gitignore

```diff
@@ -42,3 +42,5 @@ jspm_packages
 # Build files
 /Chart.Financial.js
 /Chart.Financial.min.js
+/.vs/chartjs-chart-financial/v15/.suo
```

Comment:
Actually. This is part of "4. date time formatting was added" which is already committed. Can you drop that commit and create a new PR without it?